tests/conftest.py

```diff
@@ -29,7 +29,8 @@ def pytest_configure():
             "wagtail.search",
             "wagtail.admin",
             "wagtail",
-            "wagtail.contrib.modeladmin",
+            # "wagtail_modeladmin",          # if Wagtail >=5.1; Don't repeat if it's there already
+            "wagtail.contrib.modeladmin",    # if Wagtail <5.1;  Don't repeat if it's there already
```

Comment:
Ditto

Reply:
Hi @nickmoreton ,
I based it on Jazzband's update here:
https://github.com/jazzband/wagtailmenus/pull/461/files#diff-e285c1b9901cd202e89d9f37e3249a6fee524a65428895e1a45fd062cc0f9acaR22-R23

Which is a resolution for this Wagtail 5.1 upgrade consideration: [wagtail.contrib.modeladmin is deprecated](https://docs.wagtail.org/en/stable/releases/5.1.html#wagtail-contrib-modeladmin-is-deprecated)

I updated the PR to actually replace the installed app depending on the Wagtail version.
https://github.com/torchbox-forks/wagtail-2fa/pull/3/commits/b00c2b74b69947cbfc90faa2782dccaa65a8d65c